tophatmonocle/hivemind-service
# Feature/linkedin profile app

## Description / JIRA Links
Item         | Description
https://trello.com/c/un4Y8feR
https://trello.com/c/DiHL9qNH

## DevQA
### DevQA Prep
- Have sequel pro installed
- Have the hivemind docker container running `inv docker.start`

### DevQA Steps
- run `inv docker.migrate`
- Open up sequel pro and create a new connection to the Hivemind database:
    - username: localhost
    - port: 3306
    - username: root
    - password: thm2corp
- On postman, make a GET request to http://127.0.0.1:8888/linkedin_profile_url/
- Should expect to see a response "Hello World"
- Should expect to see a new entry in the linkedin_profile_url table with name="profile1" and profile_url="htts://helloworld.com"
- On postman, make a GET request to http://127.0.0.1:8888/linkedin_profile_data/
- Should expect to see a response "Hello World"
- Should expect to see a new entry in the linkedin_profile_data table with data = {"hello": "world"}
